rust/noosphere-ns/src/bin/orb-ns/cli/cli.rs

```diff
@@ -50,6 +50,8 @@ pub enum CLICommand {
         /// instead of the default bootstrap peers.
         #[arg(short, long)]
         bootstrap: Option<Vec<Multiaddr>>,
+
+        use_test_values: bool,
```

Comment:
Alternatively, we could expose individual flags for all of the DHTConfig properties (`--peer-dialing-interval 1`), though only tests need to deviate from defaults currently, and it's still possible to configure DHTConfig via a config file via `--config <FILE>`